chef/cookbooks/crowbar/recipes/stop-services-before-upgrade.rb

```diff
@@ -63,6 +63,7 @@
       fi
     done
   EOF
+  not_if { ::File.exist?("/usr/sbin/crm") }
```

Comment:
I don't like this check.  You can't safely assume that the services are controlled by Pacemaker just because the `crmsh` package is installed.  It might be installed on non-HA environments.